rfc/rfc021-vp_token-grant-type.md

```diff
@@ -177,11 +175,14 @@ To simplify processing, this specification only allows a single main scope and m
 Allowing mixing of different main scopes in a single access token request is out of scope of this specification.
 The Authorization Server MUST reject any request that contains multiple main scopes.
 
-A main scope is defined as the first scope in the scope parameter.
-It MUST be present and MUST be a string and can't contain `:`.
+The main scope MUST be the first value in the scope parameter.
```

Comment:
I still don't think we should specify how resources are presented in OAuth2 scopes in this RFC, since it depends on whatever system the Bolt is using. And it feels like scope creep for this RFC (since this one is just about server-to-server OAuth2 authentication with a VP). And it might also apply to other RFCs, e.g. OpenID4VP/SIOP?

I'd leave out this whole scope section.

Reply:
It could be part of the use-case spec